pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/participant/ParticipantWrapper.java

```diff
@@ -100,8 +97,13 @@ private void fetchAndPrepareDataByFilters(DDPInstance ddpInstance, Map<String, S
                 }
             }
         }
-        esData = elasticSearchable.getParticipantsByRangeAndFilter(ddpInstance.getParticipantIndexES(), participantWrapperPayload.getFrom(),
+        esData = elasticSearchable.getParticipantsByRangeAndFilter(participantIndexES, participantWrapperPayload.getFrom(),
                 participantWrapperPayload.getTo(), boolQueryBuilder);
+        if (StringUtils.isNotBlank(userIndexES)) {
```

Comment:
adding getting proxy data here as well @gmakharat @Anzori-Ghurtchumelia

Reply:
@SimoneMaiwald I think it will be good idea if you pull sorting branch here, because I think proxies problem is solved already on that branch.